### PR TITLE
Add a reference to the `flutter_shaders` package

### PR DESCRIPTION
**Pull request by domesticmouse**
## Presubmit checklist

- [x] This PR is marked as draft with an explanation if not meant to land until a future stable release.
- [x] This PR doesn’t contain automatically generated corrections (Grammarly or similar).
- [x] This PR follows the [Google Developer Documentation Style Guidelines](https://developers.google.com/style) — for example, it doesn’t use _i.e._ or _e.g._, and it avoids _I_ and _we_ (first person).
- [x] This PR uses [semantic line breaks](https://github.com/dart-lang/site-shared/blob/main/doc/writing-for-dart-and-flutter-websites.md#semantic-line-breaks) of 80 characters or fewer.

### CODE REVIEW

**Comment by domesticmouse** [comment]
My open question is whether we should update the text to show how to use @jonahwilliams' `flutter_shaders` package to simplify usage?

**Review by jonahwilliams — APPROVED** [review]
LGTM

**Review by jonahwilliams — COMMENTED** [review]
as far as using the package to simplify this, I would not. We need to have some documentation of the basic API, not the opinionated builder

File: src/content/ui/design/graphics/fragment-shaders.md [hunk 1]
```diff
@@ -285,8 +285,10 @@ For more information, here are a few resources.
 * [The Book of Shaders][] by Patricio Gonzalez Vivo and Jen Lowe
 * [Shader toy][], a collaborative shader playground
 * [`simple_shader`], a simple Flutter fragment shaders sample project
+* [`flutter_shaders`] a package that simplifies using fragment shaders in Flutter
```
Comment (parlough) [inline src/content/ui/design/graphics/fragment-shaders.md:288]:
```suggestion
* [`simple_shader`][], a simple Flutter fragment shaders sample project
* [`flutter_shaders`][], a package that simplifies using fragment shaders in Flutter
```

**Review by parlough — APPROVED** [review]
One suggestion for consistent Markdown formatting, but looks good to me :)

**Review by sfshaza2 — APPROVED** [review]
lgtm. (And, yes, it works without the empty square brackets, but our convention is to include them.)

**Pull request merged**